api/infrastructure/parallelcluster-api.yaml

```diff
@@ -151,6 +161,7 @@ Conditions:
     - !Not [!Condition UsePrivateVpcEndpoint]
     - !Not [!Condition UseCustomDomainAndRoute53Configuration]
   CreateApiUserRoleCondition: !Equals [!Ref CreateApiUserRole, true]
+  EnableS3FullAccessCondition: !Equals [!Ref EnableS3FullAccess, true]
```

Comment:
What about removing the suffix `Condition`? Mostly we did not use that suffix, probably because it's a bit redundant

Reply:
Done